modules/openapi-generator/src/main/resources/python/api.mustache

```diff
@@ -263,6 +280,7 @@ class {{classname}}(object):
             post_params=form_params,
             files=local_var_files,
             response_type={{#returnType}}'{{returnType}}'{{/returnType}}{{^returnType}}None{{/returnType}},  # noqa: E501
+            response_types_map=response_types_map,
```

Comment:
Rather than using two parameters for return type
- return_type
- response_types_map
how about using a single parameter like:
- return_types which is a map of status code to data type
Please see [this similar-to PR](https://github.com/OpenAPITools/openapi-generator/pull/2134/files) which was never finished

Reply:
@spacether 
I ve just removed response_type
Could you check the pr one more time ?

Reply:
Thank you for using only one parameter. This pr has failing ci testa becausethe samples need to be regenerated. Can you please regenerate samples with these commands? 

mvn clean install
bin/generate_samples

Reply:
@spacether 
I ve just regenerated